pkg/fleetautoscalers/controller.go

```diff
@@ -293,3 +324,62 @@ func (c *Controller) updateStatusUnableToScale(ctx context.Context, fas *autosca
 
 	return nil
 }
+
+// createFasThread creates a FleetAutoScaler sync routine
+func (c *Controller) createFasThread(ctx context.Context, fas *autoscalingv1.FleetAutoscaler) error {
+	key := fas.Namespace + "/" + fas.Name
+	ticker := time.NewTicker(time.Duration(fas.Spec.Sync.FixedInterval.Seconds) * time.Second)
```

Comment:
We use https://pkg.go.dev/k8s.io/apimachinery/pkg/util/clock elsewhere in our code, so that we can control time during unit tests. We should do the same here, 

See: https://github.com/googleforgames/agones/blob/main/pkg/sdkserver/sdkserver.go#L83

Then we can write some unit tests to confirm things happen in the timeframes we expect. 👍🏻

Reply:
I removed the e2e test and added a time interval check in the "create fas thread" unit tests.